querybook/webapp/redux/dataDoc/selector.ts

```diff
@@ -213,3 +213,12 @@ export const queryCellSelector = createSelector(dataDocCellsSelector, (cells) =>
             };
         })
 );
+
+export const queryDataDocFiltersSelector = createSelector(
```

Comment:
this should be under redux/board/selector.ts.

Also you probably don't need this as a selector if you extract the board select into its own component. useMemo would be better because you also need boardById (to map list id to list options)